src/classes/CRLP_AccountSkew_BATCH.cls

```diff
@@ -61,6 +61,15 @@ public class CRLP_AccountSkew_BATCH extends CRLP_Batch_Base_Skew implements Data
      * @description Scheduler execute method to support scheduling this job directly from the UI
      */
     public void execute(SchedulableContext context) {
-        Database.executeBatch(new CRLP_SkewDispatcher_BATCH(CRLP_RollupProcessingOptions.RollupType.AccountHardCredit), CRLP_SkewDispatcher_BATCH.BATCH_SIZE);
+        String className = 'CRLP_AccountSkew_BATCH';
```

Comment:
Classes are also namespaced, so this would check will fail in a namespaced org. The way to work around this is to use the following syntax to get the class name. This will return the full class name, including the namespace if there is one.
`String className = CRLP_AccountSkew_BATCH.class.name();`